### PR TITLE
Clarifying detailed steps upon new CPI release

### PR DESCRIPTION
**Pull request by NingKuang**
No need to run unit tests since the PR only changes the documents.

### Changelog

* Update the detailed steps to use application gateway

### CODE REVIEW

**Comment by NingKuang** [comment]
Please also add steps under the prerequisite section on "Update the manifest, making sure multiple routers are added and HA Proxy removed."

File: docs/advanced/application-gateway/README.md [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
-  Application Gateway allows easy integration with Azure Traffic Manager to support multi-region redirection, automatic failover, and zero-downtime maintenance. Application Gateway is also integrated with Azure Load Balancer to support scale-out and high-availability for both Internet-facing and internal-only web front ends.
+  Application Gateway allows easy integration with Azure Traffic Manager to support multi-region redirection, automatic failover, and zero-downtime maintenance.
+
+
+## Pre-requisites
 
-# 2 Pre-requisites
+* Setup a Cloud Foundry deployment
+
+* Update the manifest, making sure multiple routers are added and HA Proxy removed.
```
Comment (bingosummer) [inline docs/advanced/application-gateway/README.md]: Update the manifest, making sure multiple routers are added and HA Proxy job removed. For example, remove the `ha_proxy_z1` job.

```
- default_networks:
  - name: cf1
  instances: 1
  name: ha_proxy_z1
  networks:
  ...
```

File: docs/advanced/application-gateway/README.md [hunk 2]
```diff
@@ -41,14 +45,14 @@ Azure Application Gateway provides application-level routing and load balancing
...
   ```
 
-### 3.1.2 Manually
+#### Creating manually with scripts
 
 * [Azure CLI](./cli/create-ag.sh)
 * [Powershell](./powershell/)
 
-## 3.2 Add Cloud Foundry routers into Application Gateway's backend pool
+### 2 Assocaite Application Gateway's backend pool and Cloud Foundry routers
```
Comment (bingosummer) [inline docs/advanced/application-gateway/README.md]: Associate Application Gateway with Cloud Foundry routers

File: docs/advanced/application-gateway/README.md [hunk 3]
```diff
@@ -1,18 +1,24 @@
...
+# Integrating Application Gateway with CF on Azure
 
-To enable advanced feature like SSL termination on your Cloud Foundry network, you can integrate Azure Application gateway. This document provides steps to create and configure [Azure Application Gateway](https://azure.microsoft.com/en-us/services/application-gateway/), that will be used as load balancer with SSL offloading enabled for Cloud Foundry. We also introduce how to create the application gateway with a single certificate or multiple certificates.
+This document provides steps to create and configure [Azure Application Gateway](https://azure.microsoft.com/en-us/services/application-gateway/), and integrate it as load balancer with SSL offloading enabled for Cloud Foundry. This can be configured with either a single or multiple certificates.
 
-# 1 What is Azure Application Gateway
+## Azure Application Gateway background
+
+Azure Application Gateway provides application-level routing and load balancing services for your web front end, by replacing HaProxy with Azure Application Gateway, Cloud Foundry cusotmers gained engerprise grade scalablity and HA with following capabilities:
```
Comment (zhongyi-zhang) [inline docs/advanced/application-gateway/README.md]: typo: engerprise -> enterprise

File: docs/advanced/application-gateway/README.md [hunk 4]
```diff
@@ -66,29 +74,29 @@ Azure Application Gateway provides application-level routing and load balancing
...
   ```
   bosh deployment multiple-vm-cf.yml
   bosh deploy -n
   ```
     
-## 3.3 Configure DNS for your Cloud Foundry domain.
+### 3. Configure DNS for your Cloud Foundry domain
 
-  * For production, you need to update the DNS configuration according to the belew sample.
+  Update the DNS entries, so the Application gateway's IP is associated with CF system doamin.
```
Comment (bingosummer) [inline docs/advanced/application-gateway/README.md]: typo: doamin

**Pull request merged**